src/layout-grid.scss

```diff
@@ -85,129 +88,76 @@ $cols: 12;
 }
 
 .#{$blockRow} {
-  @include fd-reset();
-
   display: flex;
   flex-wrap: wrap;
   flex: 0 0 100%;
   position: relative;
   margin: 0 -0.25rem;
   min-width: 100%;
 
-  &--inner-flex {
-    display: flex;
+  &:first-child {
+    margin-top: -0.5rem;
   }
 
-  .#{$block} {
-    .#{$blockRow} + .#{$blockRow} {
-      margin-top: -0.5rem;
-    }
+  .#{$blockCol} {
+    padding-top: 0.5rem;
```

Comment:
could you please put values like `0.5rem`, `0.25rem`, etc. in variables. This way we can easily change and maintain them.